application-services/README.md

```diff
@@ -0,0 +1,48 @@
+# Application Service Examples
+
+## Overview
+
+This folder contains various examples of Application Services based on the App Functions SDK and Profiles that can be used with App Service Configurable. See the [Application Services](https://docs.edgexfoundry.org/1.2/microservices/application/ApplServices/.) documentation for complete details on Application services.
+
+> *Note: All custom examples have their own `go.mod` file, `Makefile` and in some cases `Dockerfile` (e.g.`simple-filter-xml`). This allows for any example to be copied to your own repository and used as a template in creating your own new custom application service.*
+
+## Build Prerequisites
+
+Please see the [edgex-go README](https://github.com/edgexfoundry/edgex-go/blob/master/README.md).
+
+## Building Examples
+
+Each `custom` example has its own `Makefile` with the `build` target. So simply run `make build` from an example's base folder when exploring that example.
+
+The top level `makefile` is designed to build all the examples under the `custom` folder. Thus the Makefile does not need to be updated when a new example is added to the `custom` folder
+
+​	run `make build` to build all examples.
+
+For simplicity, the executable created for each example is named `app-service` and placed in each examples sub-folder.
+
+## Running an Example
+
+After building examples you simply cd to the folder for the example you want to run and run the executable for that example with or without any of the supported command line options.
```

Comment:
maybe highlight 'cd' so that it reads more easily, like a command
example: simply `cd`

Reply:
Done